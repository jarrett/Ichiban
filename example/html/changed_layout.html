<p>This page's layout should change.</p>