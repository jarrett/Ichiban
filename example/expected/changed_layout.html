<!DOCTYPE html>
<html>
  <head>
    <title>Example Site</title>
  </head>
  
  <body>
    <h1>The New Header</h1>
    
    <p>This page's layout should change.</p>
  </body>
</html>